patch/llpcPatchInOutImportExport.cpp

```diff
@@ -3152,6 +3159,48 @@ Value* PatchInOutImportExport::PatchFsBuiltInInputImport(
                                       pInsertPos);
             break;
         }
+    case BuiltInSamplePosOffset:
+        {
+            // Gets the offset of sample position relative to the pixel center for the specified sample ID
+            IRBuilder<> builder(*m_pContext);
```

Comment:
Avoid use IR builder to keep codes consistent.

Reply:
As above.